### PR TITLE
Events front

### PR DESCRIPTION
**Pull request by maasencioh**
### What was done
* Update dependencies
* Add paginated query for events
* Update current queries to use the paginated version of the query
* Create a table for events
* Link the current events with the table, using a link to the selected query

### Preview
preview

### CODE REVIEW

File: addons/events/Queries/filteredEvents.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    query.$or?.push({ processors: { history: { $size: 0 } } });
+  }
+
+  if (topic) query.topic = topic;
+  if (fileId) {
+    query['data.fileId'] = fileId;
+    query['data.type'] = GqlEventDataType.FILE;
+  }
+
+  return (await Event.getCollection()).find(query);
```
Comment (targos) [inline addons/events/Queries/filteredEvents.ts:43]: Why not `Event.query()` ?

File: addons/events/Schemas/Event.graphql [hunk 2]
```diff
@@ -1,42 +1,77 @@
...
 
 type EventProcessor {
   processorId: String!
   history: [EventHistory!]!
 }
 
+# Main event type
 type Event {
-  id: String!
+  _id: String!
```
Comment (targos) [inline addons/events/Schemas/Event.graphql]: you need a `id` field for Apollo cache to work correctly.

Comment (maasencioh) [reply addons/events/Schemas/Event.graphql]: As far as I understand in the documentation I can also use _id

https://www.apollographql.com/docs/react/caching/cache-configuration/#customizing-cache-ids

File: addons/events/Queries/filteredEvents.ts (hunk 1 above)
Comment (maasencioh) [reply addons/events/Queries/filteredEvents.ts:43]: Does query return a cursor? because I use the result for counting, sorting, and skipping

Comment (targos) [reply addons/events/Queries/filteredEvents.ts:43]: If a feature is missing, please open an issue in the adonis-mongodb repository

File: addons/events/Schemas/Event.graphql (hunk 2 above)
Comment (targos) [reply addons/events/Schemas/Event.graphql]: okay, but this is inconsistent with all other types. We should use the same name everywhere.

**Review by targos — APPROVED** [review]

**Pull request merged**